.travis.yml

```diff
@@ -7,33 +8,28 @@ matrix:
   include:
   - name: "lint check"
     os: linux
-    dist: xenial
     compiler: gcc
     env: MAKE_TARGET=format-verify PATH="/usr/lib/llvm-8/bin:$PATH"
     addons:
       apt:
         sources:
-          - llvm-toolchain-xenial-8
```

Comment:
Are we dropping support for `16.04`? We do have `20.04` around the corner...

Reply:
No, there is at least one build here still using 16.04.  I just figured it makes more sense to make the default environment 18.04 with a newer gcc.  (For instance - because it is catching that gcc release test failure).